app/routes/__auth/login.ts

```diff
@@ -12,14 +12,15 @@ import type { TokenSet } from 'openid-client'
 import { generators } from 'openid-client'
 import { getOpenIDClient, oidcStorage } from './auth.server'
 import type { getUser } from './user.server'
+import { subiss } from './user.server'
 import { updateSession } from './user.server'
 
 export function userFromTokenSet(tokenSet: TokenSet): {
   user: NonNullable<Awaited<ReturnType<typeof getUser>>>
   refreshToken: string
 } {
   const claims = tokenSet.claims()
-  const sub = claims.sub
+  const subIss = subiss({ sub: claims.sub, iss: claims.iss })
```

Comment:
Would it be more convenient if we renamed this function `getSubiss` so that we don't need to do case gymnastics to avoid collisions?